src/es/cms/index.ts

```diff
@@ -23,7 +23,7 @@ export async function getCmsFromElasticSearch({
   sort,
 }: ElasticSearchArgs) {
   limit = limit || DEFAULT_LIMIT
-  from = from || DEFAULT_FROM
+  from = from || 0
```

Comment:
We can start using the fancy `??` operator here :wink:

Reply:
Updated, as we actually don't want `from` to be a parameter in the query